components/proto/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "exonum-proto"
+version = "0.12.0"
+edition = "2018"
+authors = ["The Exonum Team <exonum@bitfury.com>"]
+homepage = "https://exonum.com/"
+repository = "https://github.com/exonum/exonum"
+documentation = "https://docs.rs/exonum-derive"
+license = "Apache-2.0"
+keywords = ["protobuf", "exonum"]
+categories = ["development-tools"]
+description = "Helper macros for serialization structs in protobuf."
+
+[dependencies]
+chrono = { version = "0.4.6", features = ["serde"] }
+failure = "0.1.5"
+protobuf = { version = "2.8.1", features = ["with-serde"] }
+bit-vec = "0.6.1"
+serde = "1.0.101"
+serde_derive = "1.0.101"
+exonum-derive = { path = "../derive", version = "0.12.0"}
```

Comment:
To be honest, it's a bit weird that `exonum-proto` depends on `exonum-derive` which uses `exonum-proto` implicitly. However, I'm not sure if this lies in the scope of this PR.